src/credentials/FabricTable.h

```diff
@@ -123,7 +123,30 @@ class DLL_EXPORT FabricInfo
         }
         return mOperationalKey;
     }
-    CHIP_ERROR SetOperationalKeypair(const Crypto::P256Keypair * keyPair);
+
+    /**
+     * Sets the P256Keypair used for this fabric, transferring ownership of the
+     * key to this object by making a copy via the P256Keypair::Serialize and
+     * P256Keypair::Deserialize methods.
```

Comment:
This is not transferring ownership.  How about:
```suggestion
     * Sets the P256Keypair used for this fabric.  This will make a copy of the keypair
     * via the P256Keypair::Serialize and P256Keypair::Deserialize methods.
     *
     * The keyPair argument is safe to deallocate once this method returns.
```